gradle/tasks/docker.gradle

```diff
@@ -0,0 +1,58 @@
+buildscript {
+    repositories {
+        mavenLocal()
+        mavenCentral()
+    }
+    dependencies {
+        classpath "com.bmuschko:gradle-docker-plugin:${gradleDockerPluginVersion}"
+    }
+}
+
+apply plugin: com.bmuschko.gradle.docker.DockerRemoteApiPlugin
+
+import com.bmuschko.gradle.docker.tasks.image.DockerBuildImage
+
+task dockerBuildImageWeb(type: DockerBuildImage) {
+    group 'Docker'
+    description 'Builds the basic web server-only Docker image'
+    // Change to 'dependsOn' when we get our up-to-date sorted
+    mustRunAfter allprojects.collect { it.tasks.matching { it.name.equals('tomcatDeploy') } }
+
+    dockerFile file('docker/Dockerfile')
+    inputDir rootProject.projectDir
+    tag 'apereo/uportal'
```

Comment:
non-blocker: in the foreseeable future, folks will probably want to be able to version the deployment images.

Reply:
Yeah I think the version should be `:${uPortalVersion}`

Reply:
quickstart should definitely be `:${uPortalVersion}`.
For the CLI and Tomcat images, `:${uPortalVersion}` could be a good starting point.
I suspect that including a build counter or a sha hash would enable more flexibility, since an institution may make many changes in uPortal start between uPortal core updates.

Reply:
Ultimately I expect adopters to want a way to override both the tag and the version for the containers they build.